warrant/warrant.py

```diff
@@ -27,11 +27,12 @@ def __init__(self, obj):
         self.subject = obj["subject"]
 
     @classmethod
-    def create(cls, object_type, object_id, relation, subject):
+    def create(cls, object_type, object_id, relation, subject, policy=""):
         payload = {
             "objectType": object_type,
             "objectId": object_id,
-            "relation": relation
+            "relation": relation,
+            "policy": policy
```

Comment:
Should we be optionally passing this in the payload only if the value is not empty? Otherwise, won't we always pass an empty string policy? See the [user module](https://github.com/warrant-dev/warrant-python/blob/master/warrant/user.py#L27) for reference. Same question for the `delete` method below.

Reply:
Yeah, I think that's more correct. Effectively though, warrants created with no policy passed vs an empty string policy are treated the same so this would still work with the current implementation.